docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.1'
+
+services:
+
+  db:
+    image: postgres
+    container_name: postgres-base
+    restart: always
+    volumes:
+    - D:\Projects\Training\Data:/var/lib/postgresql/data
```

Comment:
Let's start without `volume` for now, we can improve the docker-compose config later
Since I don't have such a directory it works only on one developer machine
By default, docker will store all data in implicitly created volume and I think that will work out for both of us

Reply:
Alright, I will remove it for now :)